lib/ipmi_backend_utils.pm

```diff
@@ -262,6 +262,21 @@ sub adjust_for_ipmi_xen {
     }
 }
 
+# Set boot from pxe via ipmitool
+sub set_pxe_boot {
+    while (1) {
+        my $stdout = ipmitool('chassis bootparam get 5');
+        if ($stdout =~ m/Boot Flag Valid[\d\D]*Force PXE/) {
+            record_info("Current boot device is PXE", "Current boot device is $stdout");
+            last;
+        }
+        diag "setting boot device to pxe";
+        my $options = get_var('IPXE_UEFI') ? 'options=efiboot' : '';
```

Comment:
This check is necessary for all UEFI types of booting? Then "options=efiboot"  should be set to options when either IPXE_UEFI or  UEFI_PXE_BOOT is non-empty?

Reply:
No, I don't think so. I don't know what 'options=efiboot' do yet. when boot_from_pxe.pm is called for our uefi machines, 'options=efiboot' is not required. Is it for accessing **i**PXE menu? @b10n1k @mdoucha @czerw may know about it.

Reply:
Aha, okay. Then current check makes sense.

Reply:
@Julie-CAO see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12483  it will not set boot order via ipmi tool without it.